<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SheetIntDiagramWidget</class>
 <widget class="QWidget" name="SheetIntDiagramWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SheetIntDiagramWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_info" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QLabel" name="label_Sheet_Index">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sheet 0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Show_Sheet_In_Mesh">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>81</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>在网格中显示</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Show_SID_In_Out_Window">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>0</y>
          <width>81</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>在外部显示</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
